### PR TITLE
Bump juttle 0.7.0

### PR DESCRIPTION
**Pull request by mstemm**
Make all necessary changes to support juttle 0.7.0 for juttle-service.

@go-oleg I reviewed your commits and lgtm, can you review mine?

### CODE REVIEW

**Comment by mstemm** [comment]
The remaining failure is being addressed in https://github.com/juttle/juttle/pull/657.

File: README.md [hunk 1]
```diff
@@ -85,10 +85,7 @@ In addition, all command-line arguments other than `--config` can also be specif
 
 ### Module resolution
 
-When juttle-service resolves module references in juttle programs while creating program bundles, it searches the following locations:
-* The configured root directory.
-* The same location as the current juttle program. For example, if a program is at ``/home/user/program.juttle`` and refers to a module ``module.juttle``, juttle-service looks in ``/home/user`` for ``module.juttle``.
-* Any locations in the environment variable JUTTLE_MODULE_PATH (colon-separated list of directories).
+For information on how modules are resolved, see the "Adapters" section in the general [juttle documentation](https://github.com/juttle/juttle/blob/master/docs/concepts/programming_constructs.md) on programming constructs.
```
Comment (go-oleg) [inline README.md]: `s/Adapters/Module Resolution/` and you can link directly to that section:

https://github.com/juttle/juttle/blob/master/docs/concepts/programming_constructs.md#importing-modules

**Comment by go-oleg** [comment]
Your changes look good, once you are all set, can you let me know, I want to squash/reword some stuff before merging.

**Comment by mstemm** [comment]
All done, feel free to rebase at will.

**Pull request merged**